### PR TITLE
[PLGN-53] Basename Update SDK

### PR DESCRIPTION
**Pull request by ablakley-r7**
### Description

Describe the proposed changes:

  - Current image uses SDK version 4.7.1 resulting in incorrect status code generation on errors
  - Update to version 5 to rectify problem
  - Ignore validator failure: basename Basename as it should not be capitalised in descriptions
  - docker validator passing locally
Screenshot 2023-08-15 at 10 10 51

## PR Requirements

Developers, verify you have completed the following items by checking them off:

#### Unit Tests

Review our documentation on [generating](https://docs.rapid7.com/insightconnect/unit-test-generation) and [writing](https://docs.rapid7.com/insightconnect/unit-test-primer) plugin unit tests

- [ ] Unit tests written for any new or updated code

#### In-Product Tests

If you are an InsightConnect customer or have access to an InsightConnect instance, the following in-product tests should be done:

- [ ] Screenshot of job output with the plugin changes
- [ ] Screenshot of the changed connection, actions, or triggers input within the InsightConnect workflow builder

### Style

Review the [style guide](https://docs.rapid7.com/insightconnect/style-guide/)

- [ ] For dependencies, pin [OS package](https://docs.rapid7.com/insightconnect/style-guide/#dockerfile) and [Python package](https://docs.rapid7.com/insightconnect/style-guide/#requirements.txt) versions
- [ ] For security, set least privileged account with ``USER nobody`` in the ``Dockerfile`` when possible
- [ ] For size, use the [slim SDK images](https://docs.rapid7.com/insightconnect/sdk-guide/#sdk-guide) when possible: ``rapid7/insightconnect-python-3-38-slim-plugin:{sdk-version-num}`` and ``rapid7/insightconnect-python-3-38-plugin:{sdk-version-num}``
- [ ] For error handling, use of [PluginException](https://docs.rapid7.com/insightconnect/error-handling-in-integrations/#plugin-exceptions) and [ConnectionTestException](https://docs.rapid7.com/insightconnect/error-handling-in-integrations#connection-exceptions)
- [ ] For logging, use [self.logger](https://docs.rapid7.com/insightconnect/sdk-guide/#logging)
- [ ] For docs, use [changelog style](https://docs.rapid7.com/insightconnect/style-guide/#changelog)
- [ ] For docs, validate markdown with ``insight-plugin validate`` which calls ``icon_validate`` to lint ``help.md``

### Functional Checklist
- [ ] Work fully completed
- [ ] Functional
  - [ ] Any new actions/triggers include JSON [test files](https://docs.rapid7.com/insightconnect/style-guide/#tests) in the `tests/` directory created with `insight-plugin samples`
  - [ ] Tests should all pass unless it's a negative test. Negative tests have a naming convention of `tests/$action_bad.json`
  - [ ] Unsuccessful tests should fail by raising an exception causing the plugin to die and an object should be returned on successful test
  - [ ] Add functioning test results to PR, sanitize any output if necessary
    * Single action/trigger `insight-plugin run -T tests/example.json --debug --jq`
    * All actions/triggers shortcut `insight-plugin run -T all --debug --jq` (use PR format at end)
  - [ ] Add functioning run results to PR, sanitize any output if necessary
    * Single action/trigger `insight-plugin run -R tests/example.json --debug --jq`
    * All actions/triggers shortcut `insight-plugin run --debug --jq` (use PR format at end)

### Assessment

You must validate your work to reviewers:

1. Run `insight-plugin validate` and make sure everything passes
2. Run the assessment tool: `insight-plugin run -A`. For single action validation: `insight-plugin run tests/{file}.json -A`
3. Copy (`insight-plugin ... | pbcopy`) and paste the output in **a new post** on this PR
4. Add required screenshots from the In-Product Tests section

### CODE REVIEW

File: plugins/basename/Dockerfile [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+FROM rapid7/insightconnect-python-3-38-plugin:5
+
+LABEL organization=rapid7
 LABEL sdk=python
-LABEL type=plugin
 
-ENV SSL_CERT_FILE /etc/ssl/certs/ca-certificates.crt
-ENV SSL_CERT_DIR /etc/ssl/certs
-ENV REQUESTS_CA_BUNDLE  /etc/ssl/certs/ca-certificates.crt
+WORKDIR /python/src
```
Comment (cmcnally-r7) [inline plugins/basename/Dockerfile:7]: Do we wanna preserve these `ENV` variables? I noticed refresh tends to wipe Dockerfiles.

File: plugins/basename/help.md [hunk 2]
```diff
@@ -52,32 +45,35 @@ Example input:
...
+  
+*This plugin does not contain any custom output types.*
 
 ## Troubleshooting
 
 If the input doesn't contain a slash `/` in the path the result will be the original string unmodified.
 
 # Version History
 
+* 1.1.1 - Update SDK to version 5
```
Comment (cmcnally-r7) [inline plugins/basename/help.md:76]: Potential reword
```suggestion
* 1.1.1 - SDK: Upgrade version to `v5`
```

**Review by cmcnally-r7 — CHANGES_REQUESTED** [review]

File: plugins/basename/Dockerfile (hunk 1 above)
Comment (ablakley-r7) [reply plugins/basename/Dockerfile:7]: Unsure. I'll import and test as a custom plugin. This plugin makes no requests.

Comment (ablakley-r7) [reply plugins/basename/Dockerfile:7]: Removal doesn't appear to cause any issues.

**Review by cmcnally-r7 — APPROVED** [review]

**Review by TKuzas-R7 — APPROVED** [review]

**Pull request merged**